CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## 1.2.0-beta.2 (Unreleased)
 
 - Add `az_iot_provisioning_client_get_request_payload()` to create MQTT payload bodies during Device Provisioning.
+- This version provides new APIs to follow the IoT Plug and Play convention to implement Telemetry, Commands, Properties and Components defined in a DTDL model.
+  - To read/write properties the SDK now provides functions to produce the right payload for components, as shown in the header sdk/inc/azure/iot/az_iot_hub_client_properties.h.
```

Comment:
I made this fix in https://github.com/Azure/azure-sdk-for-c/pull/1899 if that's ok. @jspaith are you ok with dropping `sdk/inc` from the path?

The header path the customer needs to include would be as follows:
```C
#include <azure/iot/az_iot_hub_client_properties.h>
```

That said, they wouldn't need to explicitly include it if they are already including the top-level `az_iot.h`

```suggestion
  - To read/write properties, the SDK now provides functions to produce the right payload for components, as shown in the header `azure/iot/az_iot_hub_client_properties.h`.
```

Reply:
Yup I'm cool with both of these changes thanks for piggy backing into #1899.